docs/en/observability/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -177,6 +223,15 @@ For `elasticsearch` the following arguments are supported:
   * `args.batch_max_bytes`: (Optional) Maximum size in bytes to send in a single bulk request. Default value: 10485760 (10MB).
   * `args.ssl_assert_fingerprint`: (Optional) SSL fingerprint for self-signed SSL certificate on HTTPS transport.
 
+For `logstash` the following arguments are supported:
+
+  * `args.logstash_url` URL of elasticsearch endpoint in the format `http(s)://host:port`
```

Comment:
```suggestion
  * `args.logstash_url`: URL of {ls} endpoint in the format `http(s)://host:port`
```

Reply:
Shouldn't this say Logstash endpoint? I'm using the shared attribute [defined in this file](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc#L239) to resolve the product name. (Of course we can use lowercase `logstash` when referring to the target output type.)